docs/deployment/prerequisites.md

```diff
@@ -0,0 +1,42 @@
+# Prerequisites
+
+This guide assumes that you have:
+
+1. Installed the following tools on the client machine
+    - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) - A command line tool for interacting with AWS services.
+    - [eksctl](https://eksctl.io/introduction/#installation) - A command line tool for working with EKS clusters.
+    - [kubectl](https://kubernetes.io/docs/tasks/tools) - A command line tool for working with Kubernetes clusters.
+    - [yq](https://mikefarah.gitbook.io/yq) - A command line tool for YAML processing. (For Linux environments, use the [wget plain binary installation](https://github.com/mikefarah/yq/#install))
+    - [jq](https://stedolan.github.io/jq/download/) - A command line tool for processing JSON.
+    - [kustomize version 3.2.0](https://github.com/kubernetes-sigs/kustomize/releases/tag/v3.2.0) - A command line tool to customize Kubernetes objects through a kustomization file.
+      - :warning: Kubeflow is not compatible with the latest versions of of kustomize 4.x. This is due to changes in the order resources are sorted and printed. Please see [kubernetes-sigs/kustomize#3794](https://github.com/kubernetes-sigs/kustomize/issues/3794) and [kubeflow/manifests#1797](https://github.com/kubeflow/manifests/issues/1797). We know this is not ideal and are working with the upstream kustomize team to add support for the latest versions of kustomize as soon as we can.
+   - [python](https://www.python.org/downloads/) - A programming language used for automated installation scripts.
+   - [pip](https://pip.pypa.io/en/stable/installation/) - A package installer for python.
+   
+1. Created an EKS cluster
+    - If you do not have an existing cluster, run the following command to create an EKS cluster. More details about cluster creation via `eksctl` can be found [here](https://eksctl.io/usage/creating-and-managing-clusters/).
```

Comment:
add another point

- Various controllers in Kubeflow deployment use [IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)(IRSA). An OIDC provider must exist for your cluster to use IRSA. 

Add a note
- **Note:** If you are using an existing cluster, Create an OIDC provider and associate it with for your EKS cluster by running the following command:
```
eksctl utils associate-iam-oidc-provider --cluster ${CLUSTER_NAME} \
--region ${CLUSTER_REGION} --approve
```